packages/synapse-react-client/src/lib/style/components/_annotations.scss

```diff
@@ -1,156 +1,154 @@
 @use '../abstracts/variables' as SRC;
 
-.bootstrap-4-backport {
```

Comment:
Just remove this outer .bootstrap-4-backport class which was unnecessary (and broke styling when I removed it from the modal)